src/graphql/blocks.ts

```diff
@@ -65,6 +88,15 @@ async function tsToBlockNum(network, ts) {
   return to.number;
 }
 
+async function tsToBlockNum(network: string, ts: number) {
+  try {
+    return tsToBlockWithApi(network, ts);
+  } catch (error: any) {
+    console.log(error.message);
```

Comment:
Should add more details here
```suggestion
    console.log('Failed to query block with API', error.message);
```